distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1376,3 +1376,32 @@ This is the most popular format for access logs and can be parsed by many web se
 http://team.ops4j.org/wiki/display/paxweb/Advanced+Jetty+Configuration[Advanced Jetty Configuration]
 http://wiki.eclipse.org/Jetty/Tutorial/RequestLog[Jetty Request Log Tutorial]
 
+===== Using the LogViewer
+
+Monitor system (Karaf) logs with the *LogViewer*, a convenient "viewing portal" for incoming logs.
```

Comment:
I'm a little confused by the amount of references to Karaf throughout this section. I don't know if we should call it out as much. I think `system` is more appropriate, but open to other thoughts on it.